### PR TITLE
Added closure to ontology

### PR DESCRIPTION
**Pull request by oblodgett**

### CODE REVIEW

File: src/main/java/org/alliancegenome/curation_api/services/helpers/GenericOntologyLoadHelper.java [hunk 1]
```diff
@@ -118,19 +118,19 @@ public T traverse(OWLClass parent, int depth, ArrayList<String> requiredNamespac
...
+
 			if(
-				(
-					(termParent.getNamespace() != null && requiredNamespaces.contains(termParent.getNamespace())) ||	
-					(config.getLoadOnlyIRIPrefix() != null && parent.getIRI().getShortForm().startsWith(config.getLoadOnlyIRIPrefix() + "_"))
-				) && (
-					!config.getIgnoreEntitiesWithChebiXref() || !hasChebiXref(termParent)
-				)
-			) {
+					(
```
Comment (markquintontulloch) [inline src/main/java/org/alliancegenome/curation_api/services/helpers/GenericOntologyLoadHelper.java]: Spacing looks a little funky here

**Review by markquintontulloch — APPROVED** [review]

File: src/main/java/org/alliancegenome/curation_api/services/helpers/GenericOntologyLoadHelper.java (hunk 1 above)
Comment (oblodgett) [reply src/main/java/org/alliancegenome/curation_api/services/helpers/GenericOntologyLoadHelper.java]: Thank eclipse for this one... I will fix it.

Comment (oblodgett) [reply src/main/java/org/alliancegenome/curation_api/services/helpers/GenericOntologyLoadHelper.java]: Take another look I fixed it.

**Comment by oblodgett** [comment]
@markquintontulloch this will have more work as in a migration and a change to the variable names to include "is_a"

**Comment by oblodgett** [comment]
@markquintontulloch I do believe I got the is_a relationships now

**Pull request merged**